plugin/src/main/java/jenkins/plugins/openstack/compute/JCloudsSlaveTemplate.java

```diff
@@ -122,6 +122,8 @@ private Object readResolve() {
                 lf = new LauncherFactory.SSH(credentialsId);
             } else if("JNLP".equals(slaveType)) {
                 lf = LauncherFactory.JNLP.JNLP;
+            } else if("KAFKA".equals(slaveType)) {
```

Comment:
There is no need to add these into `readResolve` methods. They are only used to migrate old config to new one. And since kafka is just being added, it does not exist in user configs.